CMakeLists.txt

```diff
@@ -57,3 +57,8 @@ add_custom_target(codespell COMMAND
   -S export-edl.js,CuteLogger,drmingw,node_modules,moc_*,index.theme,*.ts,three.min.js,three.js,jquery.js,rangy-*.js,Makefile,shotcut.pro,*.desktop
 )
 
+add_custom_target(qmlformat COMMAND
+  bash -c [[for f in $(find . -name *.qml); do ~/Qt/5.15.2/gcc_64/bin/qmlformat -i ${f}; done]]
```

Comment:
There must be a better way to get this path. If not from a CMake variable, use environment variable `$QTDIR/bin/qmlformat`. Also, in `src/CMakeLists.txt` there is a line `file(GLOB_RECURSE QML_SRC "qml/*")` that finds all the QML sources instead of relying on bash and find, which is not going to work on Windows.

Reply:
I am willing to improve this (if you have not done it already). I had mostly posted this for discussion and consideration. If you think we will go forward, I'll spiff this up.

Reply:
Done